src/website/webview/templates/components/maintainers_list.html

```diff
@@ -6,9 +6,7 @@
     <ul>
       {% for maintainer in maintainers %}
         <li class="maintainer">
-          <a class="github-id" href="https://github.com/{{ maintainer.github | urlencode }}">
-            @{{ maintainer.github }}:
-          </a>
+          <a class="github-id" href="https://github.com/{{ maintainer.github | urlencode }}"> @{{ maintainer.github }}:</a>
```

Comment:
```suggestion
          <a class="github-id" href="https://github.com/{{ maintainer.github | urlencode }}"> @{{ maintainer.github }}</a>
```

Better also skip the colon, it doesn't really inform anything in the presentation